lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -320,13 +320,27 @@ export class UserAgentApplication {
     }
 
     // #endregion
+
     /**
      * Use when initiating the login process by redirecting the user's browser to the authorization endpoint.
      * @param {@link (AuthenticationParameters:type)}
      */
     loginRedirect(userRequest?: AuthenticationParameters): void {
+        // Check if userRequest is empty
+        if (!userRequest) {
+            userRequest = {};
```

Comment:
We should avoid mutating function parameters (even when they are falsey), as it can cause intended side-effects. We'll have a lint rule in the future to disallow this.

Reply:
Done.